index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles.css">
+  <title>Luis's Portfolio</title>
+</head>
+<body>
+  <div class="wrapper">
+    <nav class="flex-nav">
+        <ul>
+          <li class="logo">My Logo </li>
+          <li class="menu-item">Portfolio</li>
+          <li class="menu-item">About</li>
+          <li class="menu-item">Contact</li>
+          <li class="hamburger-menu">☰</li>
+        </ul>
+    </nav>
+
+    <div class="content">
+      <h1>
+        I'm Luis <br>Glad to see you!
+      </h1>
+      <p>I’m a software developer! I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+      <h2 class="connect">Let's connect</h2>
+      <ul class="socials">
+        <li><a href="https://twitter.com/lujimhe">
+          <img src="/resources/TwitterVector.svg" alt="Twitter logo"></a></li>
+        <li><a href="https://www.linkedin.com/in/lujim/">
+          <img src="/resources/LinkedinVector.svg" alt="Linkedin logo"></a></li>
+        <li><a href="https://medium.com/@lu.jim">
+          <img src="/resources/MediumVector.svg" alt="Medium logo"></a></li>
+        <li><a href="https://github.com/lu-jim">
+          <img src="/resources/GithubVector.svg" alt="Github logo"></a></li>
+        <li><a href="https://angel.co">
+          <img src="/resources/AngelListVector.svg" alt="AngelList logo"></a></li>
+      </ul>
```

Comment:
- [ ] All the social icons are missing on the page, I would suggest you try using icons from [fontawesome](https://fontawesome.com/) 
![Screenshot (267)](https://user-images.githubusercontent.com/46851023/133687275-8f75518b-59d8-4a11-bb7d-b38c3a09964b.png)

Reply:
Hello! Thanks for the thorough feedback. Regarding both the background and the icons, I had used a resources folder to host them, but it seems it didn't load properly. For the logos I can certainly use fontawesome, but that would still leave the background issue. Any suggestions on how should I proceed? Would uploading the assets in another place like a public drive folder work?